src/app/content/components/Toolbar.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { SFC } from 'react';
+import styled from 'styled-components';
+import {ListOl} from 'styled-icons/fa-solid/ListOl';
+import {Search} from 'styled-icons/fa-solid/Search';
+import {Print} from 'styled-icons/fa-solid/Print';
+import theme from '../../theme';
+import Typography from '../../components/Typography';
```

Comment:
this is why you need the default export, but it would probably be better to import it like `import {bodyCopyRegularStyle} from '../../components/Typography';`